Framework/Crystal/src/FindSXPeaks.cpp

```diff
@@ -118,27 +128,29 @@ void FindSXPeaks::init() {
   // Declare
   std::vector<std::string> resolutionStrategy = {
       relativeResolutionStrategy, absoluteResolutionPeaksStrategy};
-  declareProperty("ResolutionStrategy", relativeResolutionStrategy,
-                  boost::make_shared<StringListValidator>(resolutionStrategy),
-                  "Different options for the resolution."
-                  "1. RelativeResolution: This defines a relative tolerance "
-                  "needed to avoid peak duplication in number of pixels. "
-                  "This selection will enable the Resolution property and "
-                  "disable the TofResolution, PhiResolution, ThetaResolution.\n"
-                  "1. AbsoluteResolution: This defines an absolute tolerance "
-                  "needed to avoid peak duplication in number of pixels. "
-                  "This selection will disable the Resolution property and "
-                  "enable the TofResolution, PhiResolution, "
-                  "ThetaResolution.\n");
+  declareProperty(
+      "ResolutionStrategy", relativeResolutionStrategy,
+      boost::make_shared<StringListValidator>(resolutionStrategy),
+      "Different options for the resolution."
+      "1. RelativeResolution: This defines a relative tolerance "
+      "needed to avoid peak duplication in number of pixels. "
+      "This selection will enable the Resolution property and "
+      "disable the XUnitResolution, PhiResolution, ThetaResolution.\n"
+      "1. AbsoluteResolution: This defines an absolute tolerance "
+      "needed to avoid peak duplication in number of pixels. "
+      "This selection will disable the Resolution property and "
+      "enable the XUnitResolution, PhiResolution, "
+      "ThetaResolution.\n");
 
   declareProperty(
       "Resolution", 0.01, mustBePositiveDouble,
       "Tolerance needed to avoid peak duplication in number of pixels");
 
-  declareProperty("TofResolution", 5000., mustBePositiveDouble,
-                  "Absolute tolerance in time-of-flight needed to avoid peak "
-                  "duplication in number of pixels. The values are specified "
-                  "in microseconds.");
+  declareProperty(
```

Comment:
This change will break any keyword argument usage of this call in Python. If we are okay changing this then I think `XResolution` would be slightly better.

We could deal with this by defining a specialised version of the function in Python: https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/mantid/simpleapi.py#L43 and allowing the old `TofResolution` keyword but produce a deprecation warning at the same time?

Reply:
Could do. I thought that this would be ok because this parameter was only implemented in this release cycle. So while it is in the nightly, it is not in any actual release yet.

If that is not ok then I shall do the python api fix as well.

Reply:
@samueljackson92 My apologies I didn't realise it was this release cycle. In that case changing it is okay.

Could we just rename it to `XResolution` (I think the unit part is slightly redundant)?

Reply:
@martyngigg That's ok, it was exactly obvious that was the case. Yes I shall rename the parameter.